lms/services/hubspot/service.py

```diff
@@ -84,6 +84,31 @@ def refresh_companies(self) -> None:
             ],
         )
 
+    def export_companies_contract_billables(self, date_: date):
+        """Export the contract billable numbers to HubSpot."""
+        query = (
+            self._companies_with_active_deals_query(date_)
+            .join(
+                Organization,
+                Organization.public_id == HubSpotCompany.lms_organization_id,
+            )
+            .join(OrganizationUsageReport)
+            .distinct(OrganizationUsageReport.organization_id)
+            .order_by(
+                OrganizationUsageReport.organization_id,
+                OrganizationUsageReport.until.desc(),
+            )
+        ).with_only_columns(
+            HubSpotCompany.hubspot_id,
+            OrganizationUsageReport.unique_teachers,
+            OrganizationUsageReport.unique_users,
+        )
+
+        # From the point of view of HubSpot we are creating an import
+        self._client.create_company_import(
+            self._db.execute(query).all(),
```

Comment:
This is where you should probably include `date_` in the arguments to `create_company_import()`

Reply:
Done that